rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -285,3 +285,101 @@ def test_migration_5(rotkehlchen_api_server):
     assert Path(icons_path, '_ceth_0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490_small.png').exists() is False  # noqa: E501
     assert Path(icons_path, '_ceth_0x6DEA81C8171D0bA574754EF6F8b412F2Ed88c54D_small.png').exists() is False  # noqa: E501
     assert Path(icons_path, '_ceth_0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9_small.png').exists() is False  # noqa: E501
+
+
+@pytest.mark.parametrize('data_migration_version', [None])
+@pytest.mark.parametrize('perform_migrations_at_unlock', [False])
+@pytest.mark.parametrize('perform_upgrades_at_unlock', [False])
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('perform_nodes_insertion', [False])
+def test_migration_6_default_rpc_nodes(rotkehlchen_api_server):
+    """
+    Test that the sixth data migration works when the user has not customized the nodes.
+    """
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    database = rotki.data.db
+    dir_path = Path(__file__).resolve().parent.parent.parent
+    with open(dir_path / 'data' / 'nodes_as_of_1-26-1.json', 'r') as f:
+        old_nodes_info = json.loads(f.read())
+    with open(dir_path / 'data' / 'nodes.json', 'r') as f:
+        new_nodes_info = json.loads(f.read())
+
+    old_nodes = [
+        (node['name'], node['endpoint'], node['owned'], node['weight'], node['active'], node['blockchain'])  # noqa: E501
+        for node in old_nodes_info
+    ]
+
+    with database.user_write() as write_cursor:
+        write_cursor.executemany(
+            'INSERT INTO web3_nodes (name, endpoint, owned, weight, active, blockchain) VALUES (?, ?, ?, ?, ?, ?)',  # noqa: E501
+            old_nodes,
+        )
+
+    migration_patch = patch(
+        'rotkehlchen.data_migrations.manager.MIGRATION_LIST',
+        new=[MIGRATION_LIST[5]],
+    )
+    with migration_patch:
+        DataMigrationManager(rotki).maybe_migrate_data()
```

Comment:
as mentioned in the code part, perhaps add a version where we also have all open nodes as default except for having also an own node.